ModPatches/Vanilla Arsenal/Defs/Vanilla Arsenal/Ammo.xml

```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>Ammo145x114mm_Demo</defName>
+		<label>14.5x114mm demolition</label>
+		<parent>AmmoHighCaliber</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberHighCaliber</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_145x114mm_Demo</defName>
+		<label>14.5x114mm demolition</label>
+		<ammoTypes>
+			<Ammo_145x114mm_Demo>Bullet_145x114mm_Demo</Ammo_145x114mm_Demo>

```

Comment:
Do we want the rifle to be able to fire other types of 14.5mm ammo types as well? That would give it some additional utility.

Reply:
I used purely that ammo class to try and fit it's damage around the flavour text of it being a thermobaric rifle, but also because the rifle itself is quite cheap for the size of the ammo it fires, so I felt making the only ammo source more complex to create would help balance it out